include/lemlib/chassis/chassis.hpp

```diff
@@ -236,7 +236,22 @@ class Chassis {
          * @param maxSpeed the maximum speed the robot can turn at. Default is 127
          * @param async whether the function should be run asynchronously. true by default
          */
-        void turnTo(float x, float y, int timeout, bool forwards = true, float maxSpeed = 127, bool async = true);
+        void turnToPoint(float x, float y, int timeout, bool forwards = true, float maxSpeed = 127, bool async = true);
+        /**
+         * @brief Turn the chassis so it is facing the target heading
+         *
+         * The PID logging id is "angularPID"
+         *
+         * @param targetTheta robot target heading
+         * @param timeout longest time the robot can spend moving
+         * @param radians whether the heading is in radians or degrees. false by default
+         * @param forwards whether the robot should turn to face the heading with the front of the robot. true by
+         * default
+         * @param maxSpeed the maximum speed the robot can turn at. Default is 127
+         * @param async whether the function should be run asynchronously. true by default
+         */
+        void turnToHeading(float targetTheta, int timeout, bool radians = false, bool forwards = true,
```

Comment:
there should not be an option for radians. It's highly unlikely a user will use radians, and it makes the function call more complex